operations/monitoring.rst

```diff
@@ -5,18 +5,21 @@ Monitoring
 ==========
 
 We provide a bundle overlay to simplify deploying
-`Prometheus <https://prometheus.io/>`_ and
-`Prometheus node_exporter <https://github.com/prometheus/node_exporter>`_ in
-all nodes of the cluster.
+`Prometheus <https://prometheus.io/>`_,
+`Prometheus node_exporter <https://github.com/prometheus/node_exporter>`_ and
+`slurm-exporter <https://charmhub.io/slurm-exporter>`_ to monitor the cluster
+and each individual node.
 
 
 Prometheus node_exporter
 ========================
 
+"in all nodes of the cluster"
```

Comment:
I don't understand what the context of this quote is.

Reply:
Ouch, leftover from my notes. :facepalm: